misc/discoverynode/README.md

```diff
@@ -1,8 +1,121 @@
-# Discovery
+# Discovery Node
 
-## UDMI MQTT Client
+## Notes
 
-`sudo ~/venv/bin/python3 main.py`
+`vendor` discovery family is a counter which increments every second
 
-## Unit Tests
-`~/venv/bin/python3 -m pytest tests/`
+## Standalone
+
+**TODO** - Create `bin/setup` script
+
+1.  Setup a python virtual environment and install the required dependancies
+
+    ```shell
+    cd PATH_TO_UDMI/misc/discoverynode/
+    python3 -m venv venv
+    venv/bin/python3 -m pip install -r src/requirements.txt
+    ```
+
+2. Create a JSON configuration file
+
+    Sample Clearblade configuration file
+
+    ```json
+    {
+        "mqtt": {
```

Comment:
It would be dandy if this bit matched the schema of schema/configuration_endpoint.json -- but, we can track that with some future work.  That's fundamentally what this is...

Reply:
Yup, I do agree with that and something I plan to do